specification/powerplatform/resource-manager/Microsoft.PowerPlatform/preview/2020-10-30-preview/enterprisePolicy.json

```diff
@@ -394,20 +394,58 @@
   },
   "definitions": {
     "EnterprisePolicy": {
+      "type": "object",
+      "description": "Definition of the EnterprisePolicy.",
+      "allOf": [
+        {
+          "$ref": "../../common/v1/definitions.json#/definitions/TrackedResource"
```

Comment:
Can you reference the v2 common types?

Reply:
The reason for not using the common-types is to override the description and not have location as a required property for patch operations.